src/reagent/ratom.cljs

```diff
@@ -321,7 +321,9 @@
               (and (ifn? src)
                    (not (vector? src))))
           (str "src must be a reactive atom or a function, not "
-               (pr-str src)))
+               (pr-str src)
+               "while attempting to get path: "
```

Comment:
I think this is missing space on the beginning?

Reply:
good spot, will fix.